apps/omg_child_chain/lib/omg_child_chain/fee_server.ex

```diff
@@ -56,34 +62,45 @@ defmodule OMG.ChildChain.FeeServer do
     {:ok, load_fees()}
   end
 
-  @doc """
-   Reads fee specification file if needed and updates :ets state with current fees information
-  """
+  def handle_info(:update_fee_spec, state) do
+    _ =
+      if Application.get_env(:omg_child_chain, :ignore_fees) do
+        _ = Logger.warn("Fee specs from file are ignored. Updates takes no effect.")
+      else
+        alarm = {:invalid_fee_file, Node.self(), __MODULE__}
+        _ = Alarm.set(alarm)
+
+        with :ok <- update_fee_spec(),
```

Comment:
hm... is the logic correct? you're raising the alarm before you know whether it's ok and then clearing it?